test-smart-contract/src/lib.rs

```diff
@@ -34,8 +37,7 @@ fn smartcontract_entry_point(_account_id: AccountId) {
         let nft_id = generate_new_nft_id(account.id());
         let nft_definition = AssetDefinition::store(nft_id.clone())
             .mintable_once()
-            .with_metadata(metadata)
-            .build();
```

Comment:
build is not needed anymore?

Reply:
Yes. We copied these changes from Iroha repo.